addons/one-click-studio-invites/addon.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://raw.githubusercontent.com/ScratchAddons/manifest-schema/dist/schema.json",
+  "name": "One-click studio invites",
+  "description": "TODO",
```

Comment:
Fix this then merge

Reply:
I'll take care of this

Reply:
12 days later

Reply:
@Weredime Don't worry, I'm done with exams now.